models/music_event_in_proxy.h

```diff
@@ -40,7 +40,7 @@
 namespace nest
 {
 
-/* BeginUserDocs: devices, MUSIC
+/* BeginUserDocs: device, MUSIC, spike
```

Comment:
:+1:

This is different from *continuous* in the that these proxies actually receive and send spike events from/to neurons and thus have to be listed in some index together with them.